miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/RealMiniApp.kt

```diff
@@ -28,4 +29,15 @@ internal class RealMiniApp(
             displayer.createMiniAppDisplay(basePath, appId)
         }
     }
+
+    override fun updateConfiguration(settings: MiniAppSdkConfig) {
+        miniAppDownloader.setApiClient(
+            ApiClient(
```

Comment:
Both `MiniAppDownloader` and `MiniAppInfoFetcher` would also need to update the ApiClient.
It will be good to store the ApiClient instances like it was happening for the SDK instances previously. What do you think?

Reply:
Also, we will need to add validations for the properties of `MiniAppSdkConfig` and throw some error when invalid settings are provided for.